cmd/touch/touch.go

```diff
@@ -101,3 +102,33 @@ func Touch(ctx context.Context, fsrc fs.Fs, srcFileName string) (err error) {
 	}
 	return nil
 }
+
+// Recursively touch every file in f with time t
+func TouchRecursive(ctx context.Context, f fs.Fs, t time.Time) error {
+	timeAtr := time.Now()
+	if timeAsArgument != "" {
+		layout := defaultLayout
+		if len(timeAsArgument) == len(layoutDateWithTime) {
+			layout = layoutDateWithTime
+		} else if len(timeAsArgument) > len(layoutDateWithTime) {
+			layout = layoutDateWithTimeNano
+		}
+		var timeAtrFromFlags time.Time
+		if localTime {
+			timeAtrFromFlags, err = time.ParseInLocation(layout, timeAsArgument, time.Local)
+		} else {
+			timeAtrFromFlags, err = time.Parse(layout, timeAsArgument)
+		}
+		if err != nil {
+			return errors.Wrap(err, "failed to parse date/time argument")
+		}
+		timeAtr = timeAtrFromFlags
+	}
+	return ListFn(ctx, f, func(o fs.Object) {
```

Comment:
Maybe this needs to be `operations.ListFn...`, and with an `"github.com/rclone/rclone/fs/operations"` entry in import up in the beginning of the file.

Reply:
@ncw What would you advise to do here? Just you gave me the code for the ListFn function.  https://forum.rclone.org/t/using-rclone-touch-recursively/23975/13